tests/app/storage/test_questionnaire_storage.py

```diff
@@ -4,37 +4,37 @@
 from sqlalchemy.exc import IntegrityError
 
 from app import settings
-from app.storage.questionnaire_storage import QuestionnaireStorage
+from app.storage.encrypted_questionnaire_storage import EncryptedQuestionnaireStorage
```

Comment:
This test file should be merged into `tests/app/storage/test_encrypted_questionnaire_storage.py` - although not followed consistently, there should be a one to one mapping from code to test files for unit tests.